frontend/front/src/pages/Public/Components/Button/ButtonDarkBlue.js

```diff
@@ -13,11 +13,14 @@ const ButtonDarkBklue = ({ text, to, children, externalLink }) => {
         <Button
           onClick={() => window.scrollTo(0, 0)}
           sx={{
-            width: "200px",
+            px: 2,
+            width: "auto",
+            minWidth: "200px",
             height: "50px",
             color: "var(--color-text-1)",
             background: "var(--bgColor-2)",
             borderRadius: "50px",
+            textAlign: "center",
             "&:hover": {
               backgroundColor: "var(--bgColor-2)",
             },
```

Comment:
I would stick these in a constant called something like "buttonStyle" so that you can reuse in both alternates.

Also, this was an issue before this PR, but the correct way to use a button as a link when using MUI with react-router is like this ([source](https://mui.com/material-ui/guides/routing/#button)):

```
import {Link} from 'react-router-dom'
...

<Button
  component={Link}
  to={url}>
    {children}
</Button>
```

Reply:
I think this button is supposed to scroll to the top of the page?

Reply:
We have three types of buttons, all located in the 'Button' folder:

1. **ButtonDarkBlue.js** (reusable): This button is configured to use an external link (href) and React Router DOM with to. If externalLink is true, it uses href, and if externalLink is false, it uses to.
![Screenshot 2023-10-25 at 3 30 03 PM](https://github.com/codeforboston/urban-league-heat-pump-accelerator/assets/25167383/b3b6d7ad-eecf-4ebc-a990-4465fce0e5c5)

2. **ButtonWhite.js** (reusable): This button is also reusable and serves a similar purpose. It only uses React Router DOM with `to."
![Screenshot 2023-10-25 at 3 33 01 PM](https://github.com/codeforboston/urban-league-heat-pump-accelerator/assets/25167383/647c36a5-f18c-41e9-970b-c4c9f23072f4)

3. **ButtonGetPump.js** (not reusable): Unlike the previous two buttons, this one is not reusable. It also uses React Router DOM with `to."
![Screenshot 2023-10-25 at 3 33 59 PM](https://github.com/codeforboston/urban-league-heat-pump-accelerator/assets/25167383/24239404-7548-4a19-8020-c0e288db0ca9)

P.S. If you believe there's a more appropriate name for any of these buttons, please let me know.

Reply:
About the Link. Replaced <a> tag with MuiLink from @mui/material for external links.
`import { Button, Link as MuiLink } from "@mui/material";`

Reply:
@thiagobardini, this is kind of the same thing. MUI Link is basically just `<a>` but allowing the custom MUI props like `sx`. It's not valid HTML5 because you're wrapping one interactive element `<button>` with another interactive element `<a>` ([source](https://www.freecodecamp.org/news/html-button-link-code-examples-how-to-make-html-hyperlinks-using-the-href-attribute-on-tags/)).

Actually, the button is implemented correctly in the second return statement of the component, when it is not an external link. I suggest rewriting the component with a single return statement like this:
```
return (
    <>
      <Button
        component={Link} // Make sure this is Link from 'react-router-dom'
        to={to}
        target={externalLink ? "_blank" : "_self"}
        // The onClick scroll is not necessary when opening an external link
        onClick={externalLink ? () => window.scrollTo(0, 0) : undefined} 
        sx={{
          px: 2,
          width: "auto",
          minWidth: "200px",
          height: "50px",
          color: "var(--color-text-1)",
          background: "var(--bgColor-2)",
          borderRadius: "50px",
          textAlign: "center",
          "&:hover": {
            backgroundColor: "var(--bgColor-2)",
          },
        }}
      >
        {text ? text : children}
      </Button>
    </>
  );
```

Reply:
It is implemented correctly in ButtonWhite and ButtonGetPump, but those two don't account for the case of an external link that opens with _blank.

Reply:
Yo're right! I'll change it.